packages/shared/src/graphql/search.ts

```diff
@@ -199,9 +207,13 @@ export const updateSearchData = (
 
   const updated = {
     ...previous,
-    chunks: [{ ...previous.chunks[0], ...chunk }],
+    chunks: [{ ...previous?.chunks?.[0], ...chunk }],
```

Comment:
`chunks` can be undefined when error is first thing sent like in rate limit case